modules/swagger-codegen/src/main/resources/JavaJaxRS/cxf/pom.mustache

```diff
@@ -4,6 +4,9 @@
   <artifactId>{{artifactId}}</artifactId>
   <packaging>jar</packaging>
   <name>{{artifactId}}</name>
+  {{#appDescription}}
+  <description>{{{appDescription}}}</description>
```

Comment:
@gustavoapaz I think you will need to use {{appDescription}} instead to use the HTML-escaped value in handling special characters like <, > in the description.